pkgs/top-level/all-packages.nix

```diff
@@ -10443,24 +10443,15 @@ in
 
   any-nix-shell = callPackage ../shells/any-nix-shell { };
 
-  bash = lowPrio (callPackage ../shells/bash/4.4.nix {
+  bash = lowPrio (callPackage ../shells/bash/5.1.nix {
```

Comment:
Can we keep bash_4?

Reply:
Sure, what are you thinking about ? Just for people to be able to use?
under `bash4` then?

Reply:
usually its `name_X` when extra versions are packaged for extra use (either older or newer than the main package)